CHANGES.rst

```diff
@@ -45,8 +45,16 @@ alma
 Infrastructure, Utility and Other Changes and Additions
 -------------------------------------------------------
 
+0.4.1 (2020-02-07)
+================
```

Comment:
Could you make this change?  We didn't release 0.4.1 on Feb 7.  Let us set the release time later.
```suggestion
0.4.1 (unreleased)
==================
```

Reply:
also, you should be able to make this change through the github interface